cypress/e2e/login.cy.js

```diff
@@ -2,10 +2,40 @@
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+  cy.visit('/login');
   });
 
-  it('', () => {
-    
+  it('should login user with registerd credentials', () => {
+
+  cy.get('[name="username"]').click().type('tomsmith');
```

Comment:
```suggestion
  cy.get('[name="username"]').type('tomsmith');
```
no need to click the field, it's already in focus after `cy.get`